src/main/kotlin/no/nav/syfo/dialogmote/database/domain/PMotedeltakerBehandler.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.dialogmote.database.domain
+
+import no.nav.syfo.dialogmote.domain.*
+import java.time.LocalDateTime
+import java.util.UUID
+
+data class PMotedeltakerBehandler(
+    val id: Int,
+    val uuid: UUID,
+    val createdAt: LocalDateTime,
+    val updatedAt: LocalDateTime,
+    val moteId: Int,
+    val behandler_ref: String,
+    val behandler_navn: String,
+    val behandler_kontor: String,
+    val behandler_type: String,
```

Comment:
Understrekt brukes kun i database, i Kotlin-koden er det camelCase som gjelder, e.g: `behandlerRef` og tilsvarende for resten av navnene i PR

Reply:
Ja, det er mer konsistent med hvordan koden ellers er. Fikset.